cmd/gnmi_cli/gnmi_cli.go

```diff
@@ -64,7 +64,9 @@ var (
 	queryType   = flag.String("query_type", client.Once.String(), "Type of result, one of: (o, once, p, polling, s, streaming).")
 	queryAddr   = flags.NewStringList(&q.Addrs, nil)
 
-	reqProto = flag.String("proto", "", "Text proto for gNMI request.")
+	reqProto     = flag.String("proto", "", "Text proto for gNMI request.")
+	encodingType = flag.String("encodingType", "JSON", "Request Encoding Type")
+	tlsDisabled  = flag.Bool("tlsDisabled", false, "When set, the TLS will be disabled for gRPC connection")
```

Comment:
should say that `caCert, clientCert & clientKey` will be ignored

Reply:
Done